common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -76,4 +76,14 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-commercial-consent-global",
+    "Test the TCF globally",
+    owners = Seq(Owner.withGithub("frankie297")),
```

Comment:
Are you happy to be the owner given that you'll have moved off the team by then? Is it better to hand this to someone that will be working in commercial dev next quarter?

Reply:
I am updating this switch to be owned by `Commercial`. 👍 😸